generators/entity-client4/templates/ionic/src/app/pages/entities/_entity-edit.html.ejs

```diff
@@ -54,7 +54,7 @@ _%>
             _%>
             <ion-item lines='none'>
                 <%_ if (fields[idx].fieldIsEnum) { _%>
-                <ion-label> {{ '<%=fieldNameCapitalized _%>' | translate }}:</ion-label>
+                <ion-label> {{ '<%=fieldNameCapitalized _%>' | translate }}:</b></ion-label>
```

Comment:
I know this is work-in-progress, but this line looks like it is missing the opening `<b>`

Reply:
Thanks @ruddell, Fixed :-) change to floating again. It looks nicer :-) then bold label